exporter.py

```diff
@@ -66,9 +68,11 @@ def run_prometheus_server(port, collectors, *args):
     thread = Thread(target=InventoryBuilder, args=(options.atlas,))
     thread.start()
     collectors = [
-                HostSystemStatsCollector(),
```

Comment:
please leave our other collectors in place ;)

Reply:
This has been also uncommented